applications/nrf_desktop/configuration/nrf52kbd_nrf52832/prj_nob0.conf

```diff
@@ -185,13 +184,22 @@ CONFIG_NFCT_PINS_AS_GPIOS=y
 
 CONFIG_ASSERT=y
 CONFIG_ASSERT_LEVEL=2
-CONFIG_ASSERT_VERBOSE=n
 
-################################################################################
-# Bootloader Configuration
-
-CONFIG_SECURE_BOOT=y
-CONFIG_BUILD_S1_VARIANT=y
-CONFIG_FW_INFO=y
-CONFIG_FW_INFO_FIRMWARE_VERSION=1
-CONFIG_SB_SIGNING_KEY_FILE="configuration/nrf52kbd_nrf52832/b0_private.pem"
+CONFIG_LOG=y
```

Comment:
The configuration specific fact for this one is that we enabled logs. Maybe we could call it `prj_logs.conf` (bootloader is not enabled because of memory limitations)

Reply:
changed to prj_nob0_logs.conf